src/mpi/pt2pt/irecv.c

```diff
@@ -127,6 +127,8 @@ int MPI_Irecv(void *buf, int count, MPI_Datatype datatype, int source,
 
     mpi_errno = MPID_Irecv(buf, count, datatype, source, tag, comm_ptr,
                            MPIR_CONTEXT_INTRA_PT2PT, &request_ptr);
+    /* If the error is cannot allocate request, goto fail before accessing the request */
+    if (mpi_errno != MPI_SUCCESS) goto fn_fail;
```

Comment:
Can you run the `code-cleanup.sh` script on this?

Reply:
Sure. Will run it once merged with recent changes.